source/linux/background-managers/swaybg.js

```diff
@@ -0,0 +1,25 @@
+import childProcess from 'node:child_process';
+import {commandExists} from '../util.js';
+
+export async function isAvailable() {
+	return commandExists('swaybg');
+}
+
+export async function set(imagePath) {
+	const spawn = (
+		cmd,
+		args,
+	) => new Promise((resolve, reject) => {
+		const cp = childProcess.spawn(cmd, args);
+		cp.stderr.on('data', data => {
+			if (data.includes('Failed to load image')) {
+				cp.kill('SIGINT');
+				reject();
```

Comment:
You need to preserve the error message. Promise rejecting without anything is very useless to an end-user using this library.

Reply:
I recovered the error message, but I don't understand why I need this error message if I didn't catch it.